.github/workflows/win.yml

```diff
@@ -1,6 +1,12 @@
 name: Windows CI
 
-on: [push, pull_request]
+on:
+  # Trigger the workflow on push on the master branch,
+  # or for any pull request
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
If I understand it correctly, the CI will run only after the PR has been accepted ? Unless our original branch is called master, but I never do that (I don't like 'master has been merged onto master') :)

Reply:
nope, it runs the CI if the destination branch is master! Anyway I removed this condition, so that any PR now triggers the CI.